packages/editor/src/browser/editor-preferences.ts

```diff
@@ -10,36 +10,34 @@ import {
     createPreferenceProxy,
     PreferenceProxy,
     PreferenceService,
+    PreferenceContribution,
+    PreferenceSchema
 } from '@theia/preferences/lib/common';
 
-export const EditorConfigSchema = {
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "title": "Editor configuration Scheme",
-    "type": "object",
-    "properties": {
+export const editorConfigSchema: PreferenceSchema = {
```

Comment:
I think I would keep Configuration, it makes sense (with further thinking...). So Configuration and preferences, but no more config/fs/prefs got it :)